packages/core/core/src/AssetGraphBuilder.js

```diff
@@ -155,6 +156,12 @@ export default class AssetGraphBuilder extends EventEmitter {
     let cacheEntry = await this.runTransform(req);
     let time = Date.now() - start;
 
+    report({
```

Comment:
Not sure if this is the right place and probably not a proper phase name either

Reply:
The other reporting about transforming happens inside the TransformerRunner. Should this one go there too?

Reply:
I guess that would work although I'm kinda more leaning towards the other approach of keeping the map of changedassets in core and just sending it to all reporters on buildSuccess